soh/include/z64save.h

```diff
@@ -322,6 +321,10 @@ typedef struct {
     // #endregion
 } SaveContext; // size = 0x1428
 
+#define IS_MASTER_QUEST(saveContext) ((saveContext).questId == 1)
+#define IS_RANDO(saveContext) ((saveContext).questId == 2)
+#define IS_BOSS_RUSH(saveContext) ((saveContext).questId == 3)
```

Comment:
could we do

```suggestion
typedef enum {
  QUEST_ORIGINAL = 0,
  QUEST_MQ = 1,
  QUEST_RANDOMIZER = 2,
  QUEST_BOSSRUSH = 3,
} QuestId;

#define IS_MASTER_QUEST(saveContext) ((saveContext).questId == QUEST_MQ)
#define IS_RANDO(saveContext) ((saveContext).questId == QUEST_RANDOMIZER)
#define IS_BOSS_RUSH(saveContext) ((saveContext).questId == QUEST_BOSSRUSH)
```

to make this less magic-numbery?

Reply:
just noticed proxy suggested macros, that works too

Reply:
Done.